chain/chunks/README.md

```diff
@@ -0,0 +1,37 @@
+# near-chunks
+
+This crate cotains functions to handle chunks. In NEAR - the block consists of multiple chunks - at most one per shard.
+
+When chunk is created, the creator encodes its contents using Reed Solomon encoding (ErasureCoding) and adds cross-shard receipts - creating PartialEncodedChunks, that are later sent to all the validators (each validator gets a subset of them). This is done for data availability reasons (so that we need only a part of the validators to reconstruct the whole chunk). You can read more about it in the Nightshade paper.
```

Comment:
```suggestion
When a chunk is created, the creator encodes its contents using Reed Solomon encoding (ErasureCoding) and adds cross-shard receipts - creating PartialEncodedChunks that are later sent to all the validators (each validator gets a subset of them). This is done for data availability reasons (so that we need only a part of the validators to reconstruct the whole chunk). You can read more about it in the Nightshade paper.
```
Could we link the nightshade paper here?